source/initial_conditions/S40RTS_perturbation.cc

```diff
@@ -625,6 +631,10 @@ namespace aspect
                              Patterns::Double (0),
                              "The reference temperature that is perturbed by the spherical "
                              "harmonic functions. Only used in incompressible models.");
+          prm.declare_entry ("Remove temperature heterogeneity down to specified depth", "-10",
+                             Patterns::Double (),
+                             "This will set the heterogeneity prescribed by S20RTS or S40RTS to zero "
+                             "down to the specified depth.");
```

Comment:
Can you state the units of this parameter (I assume meters)? Also, what does the default value of -10 mean: a *height* of ten meters? If so, why?

If you want the default to be to not zero out any heterogeneities, then setting it to some very large negative value would be the right thing. Zero would also be a valid value.